modules/oauth/src/main/java/org/mule/module/oauth2/internal/authorizationcode/DefaultAuthorizationCodeGrantType.java

```diff
@@ -169,8 +170,7 @@ public void initialise() throws InitialisationException
             resourceOwnerIdEvaluator.initialize(muleContext.getExpressionManager());
             if (externalRedirectionUrl == null)
             {
-                String url = System.getProperty(EXTERNAL_REDIRECT_URL_PROPERTY);
```

Comment:
I would still support the system property buy fail if both, the system property and the externalRedirectionUrl attribute are defined.